plugins/GSdx/GLLoader.cpp

```diff
@@ -482,4 +490,9 @@ namespace GLLoader {
 
 		s_first_load = false;
 	}
+	
+	#define PRINT_ONCE(stream, ...)	do { \
```

Comment:
I'm getting a compile error tho , I'll need some help.
`Error C2664 'int printf(const char *const ,...)': cannot convert argument 1 from 'FILE *' to 'const char *const `'